tests/coprocessor/test_select.rs

```diff
@@ -266,6 +267,7 @@ fn test_select() {
 
 #[test]
 fn test_group_by() {
+    init_log();
```

Comment:
There is no official support for this yet. And I find some third party crates, but using them requires a lot of changes.